Bottom: 30708e93c06733302f42c8957ef0a1c230e5ae79
Top:    f49a2f2bbf20a42b97670fbb2b14edad8462aaa5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-07-05 16:42:51 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
